src/main/resources/avro/common.avdl

```diff
@@ -5,44 +5,45 @@ not have any methods; it is merely a library of types.
 
 ##Sequences, Segments, and References
 
-A sequence is a component in a sequence graph, optionally joined at its ends to
-other sequences. Sequences are not directly retrievable through the API, but
-ranges of their bases can be downloaded.
-
-A `Segment` is a range on a sequence, possibly including the joins at the
-sequence's ends. It does not include base data. When you ask about the novel
-sequences that come in a `VariantSet`, you will get back `Segment`s covering
-them. A `Path` through a sequence graph is a list of `Segment`s.
-
-A `Reference` (defined in references.avdl) is an object that marks a sequence as
-part of a `ReferenceSet`'s sequence graph. It contains a `Segment` covering the
-sequence it belongs to.
+A `Sequence` is a component in a sequence graph, optionally joined to other
+`Sequence`s via `Join`s. It has to have length at least 1, but does not contain
```

Comment:
Previously `Segment`s were used to represent deletions as well. In this version of the graph model, a deletion, transposition, or other base-less change is effected through use of pure `Join`s, with no `Segment`s involved. Thus the requirement for length > 0.